data/members/sixscape.yaml

```
@@ -0,0 +1,71 @@
+id: sixscape
+name: Sixscape Communications
+slogan: Cryptographic Authentication & Identity
+website: https://www.sixscape.com/
+
+# Short description, longer content can be placed in `content/members/`
+description: Sixscape is dedicated to making Cryptographic Authentication and Secure Messaging Simple, to enable secure digital transformation, IoT deployment and allow 5G to be fully and securely exploited.
+
+# membership
+memberSince: 2021-07-14
+memberType: Vendor
+workingGroups:
+
+# sponsor
+#sponsor:
+#  level:
+
+# Blog posts
+blog: 
+  url:
+  feed:
+  language: en_US
+
+# Press Releases
+press: 
+  url: https://sixscape.com/news/
+  feed: 
+  language: en_US
+
+# Careers
+careers: 
+  url: https://sixscape.com/careers/
+  feed: 
+  language: en_US
+
+# Social media
+social:
+  twitter: https://twitter.com/sixscape_comm?lang=en
```

Remove optional parameters
```suggestion
  twitter: https://twitter.com/sixscape_comm
```